docs/format.md

```diff
@@ -1,7 +1,7 @@
 Description of YAML format
 ==========================
 
-Specification version: 1.0
```

Comment:
Also, this is not the version of the _content_. It is the version of the format. The version of the content is the version of simphony-metadata.